mscore/file.cpp

```diff
@@ -1209,9 +1209,27 @@ QString MuseScore::getFotoFilename(QString& filter, QString* selectedFilter)
       // setup side bar urls
       saveImageDialog->setSidebarUrls(urls);
 
+      // set the current score's name as the default name for saved captures
+      QString scoreName = cs->masterScore()->fileInfo()->completeBaseName();
+      QString name = createDefaultFileName(scoreName);
+      QString fname = QString("%1/%2").arg(defaultPath).arg(name);
+      QFileInfo myCapture(fname);
+      if (myCapture.isRelative())
+            myCapture.setFile(QDir::home(), fname);
+      QString myCapturePath = myCapture.absoluteFilePath();
+      // if no saves were made for current score, then use the score's name as default
+      if(cs->masterScore()->savedCapture() == false)
+            saveImageDialog->selectFile(myCapturePath);
+      else
+            saveImageDialog->selectFile(lastSaveCaptureName);
```

Comment:
at this point, you computed myCapture path and in the end, you will not use it. Could you change that?